chrome/views/gmailView.js

```diff
@@ -25,12 +25,13 @@ GmailToTrello.GmailView = function(parent) {
         emailBody: '.adP:first', // Was: '.adO:first'
         emailAttachments: '.aZo', // Was: '.aQy',
         emailThreadID: '.a3s.aXjCH',
-        viewport: '.aeJ:first',
+        viewport: '[role="main"]',
```

Comment:
I'm still not sure this is working perfectly. Because we're running parseData immediately onDetection, the selectors don't seem to always get the current page. When I navigated from a thread to a landing page in Gmail, .aej:first was getting the thread page, so the button was capturing the threadCards for the previous thread and displaying them on the landing page. This seems to fix it, although I think there may still be a race condition in here somewhere...